core-blocks/playlist/index.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Server-side rendering of the `core/playlist` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the `core/playlist` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Playlist output. Empty string if the passed type is unsupported.
+ */
+function render_block_core_playlist( $attributes ) {
+	$attributes['ids'] = json_decode( $attributes['ids'] );
```

Comment:
Yeah my memory is a little hazy on the reasons behind this but I remember it being due to an error thrown by the REST API.  I'll look into it and document if I can find why this was.

Reply:
From memory I think it's because Gutenberg isn't clever enough to parse/serialise the array as `ids[]=1&ids[]=2&ids[]=3` which is what PHP expects and generates.